.github/workflows/release.yaml

```diff
@@ -65,3 +65,37 @@ jobs:
         asset_path: ./src/github.com/${{ github.repository }}/release.yaml
         asset_name: release.yaml
         asset_content_type: text/plain
+
+  kn-vsphere:
+    name: Release vSphere Knative plugin
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: ${{ github.workspace }}
+      GO111MODULE: off
+
+    steps:
+      - name: Check out code onto GOPATH
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          path: ./src/github.com/${{ github.repository }}
+
+      - name: Unshallow
+        run: |
+          cd ./src/github.com/${{ github.repository }}
+          git fetch --prune --unshallow
+
+      - name: Set up Go 1.14.x
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14.x
+
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v1
+        with:
+          version: latest
+          args: release --rm-dist
+          workdir: ./src/github.com/${{ github.repository }}/plugins/vsphere
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
One lurking concern I have is that we'll end up with two releases from these two jobs.  Something we'll want to check on post-submit.

Reply:
is there a way to tell `ko` to ignore `./plugins/*`? That way, plugins would be released separately.

Reply:
I think the current release script for the whole repo can be ported to goreleaser, as a build step preceding the plugin build step. I'll have a crack at it tomorrow.

Reply:
So I tried using gorelease for the ko release but it's a dead-end.
What I'm doing instead is have the ko release job depend on the plugin release job.
The ko release job then piggybacks on the release created by goreleaser to upload the `release.yaml`.